packages/clay-label/src/index.tsx

```diff
@@ -59,37 +78,25 @@ interface IProps
 	 */
 	large?: boolean;
 
-	/**
-	 * Attributes to apply to the inner label element
-	 */
-	innerElementProps?: React.HTMLAttributes<
-		HTMLAnchorElement | HTMLSpanElement
-	>;
-
 	/**
 	 * Path to the location of the spritemap resource used for Icon.
 	 */
 	spritemap?: string;
 }
 
-const ClayLabel = React.forwardRef<HTMLAnchorElement | HTMLSpanElement, IProps>(
+const ClayLabelBase = React.forwardRef<HTMLSpanElement, IBaseProps>(
```

Comment:
I think we went back to the name `Base` 😂, is a dead end!

Reply:
haha yeah I wasn't sure what else to call it. I'll update to be similar to our other components so that we avoid confusion.

Reply:
haha no problem. I think we can keep the `Base`.